README.md

```diff
@@ -1,48 +1,117 @@
-# WIP :construction: :warning:
-We are currently working on a 2nd version of WebApiClient which will be based on Refit and will support GraphQL. Please look at the [issues](https://github.com/Xablu/Xablu.WebApiClient/issues) section for more details.
+# Xablu.WebApiClient   ![xablu logo ](/readmecontent/xablu_logo.png "Xablu")
 
-# Xablu.WebApiClient
-The Xablu WebApiClient is a C# HTTP library which aims to simplify consuming of Web API services in .NET projects.
+The Xablu WebApiClient is a C# HTTP library which aims to simplify consuming of Web API services in .NET projects. 
 
-## Setup
-The Xablu.WebApiClient is written according to the Xamarin Plugin philosophy. Meaning you can simply add the Xablu.WebApiClient package through 
-[NuGet](https://www.nuget.org/packages/Xablu.WebApiClient). Install the NuGet package into your shared .NET Standard project and ALL Client projects.
+> :construction: :warning: Already using this library? 
+> We have been working on a 2nd version of WebApiClient which is based on Refit and will support GraphQL. This version has may new features. Beware, upgrading from version 1 to version 2 has some breaking changes since it’s not backwards compatible. 
 
-* NuGet: [Xablu.WebApiClient](https://www.nuget.org/packages/Xablu.WebApiClient)
-* `PM> Install-Package Xablu.WebApiClient`
+## Table of contents  
+1. [How Xablu.WebApiClient Works] (#howto)
```

Comment:
I think the spaces between `]` and `(` need to be removed in this list, otherwise links are not shown correctly

Reply:
done